src/index.html

```diff
@@ -15,8 +15,107 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="img-link"
+        href="./index.html"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo-img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="list-item apple">
+            <a
+              href="./"
+              class="nav-link is-active apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="list-item samsung">
+            <a
+              href="./"
+              class="nav-link samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="list-item smartphones">
+            <a
+              href="./"
+              class="nav-link smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="list-item laptops">
+            <a
+              href="./"
+              class="nav-link laptops"
```

Comment:
Avoid using class names that represent styles or tag names. The class name 'laptops' does not represent the meaning of the content. Refer to the checklist: `Class names represent the meaning of the content (not the styles or tag names)`.